conda/activate.py

```diff
@@ -99,6 +105,8 @@ def __init__(self, shell, arguments=None):
             self.export_var_tmpl = '@SET "%s=%s"'
             self.run_script_tmpl = '@CALL "%s"'
 
+            self.deactivatepost_tmpl = """@CALL %_CONDA_EXE% shell.cmd.exe deactivatepost"""
```

Comment:
For `cmd.exe`, I think  we'd need the monstrosity of
https://github.com/conda/conda/blob/4.5.0rc0/conda/shell/Library/bin/conda.bat#L45
https://github.com/conda/conda/blob/4.5.0rc0/conda/shell/Library/bin/conda.bat#L50-L52
(Maybe with a slightly different variable name for `_TEMP_SCRIPT_PATH` to avoid collision?)

Reply:
wow I didn't even process how ugly the `cmd.exe` version was...